src/index.ts

```diff
@@ -131,16 +131,16 @@ async function openapiTS(
       let comment: string | undefined;
       switch (subschema.hint) {
         case "OpenAPI3": {
-          const subschemaTypes = transformSchema(subschema.schema, { ...ctx, indentLv });
+          const subschemaTypes = transformSchema(subschema.schema, { ...ctx, indentLv: indentLv + 1 });
           if (!Object.keys(subschemaTypes).length) break;
-          output.push(indent(`${key}: {`, 1), "");
+          output.push(indent(`${key}: {`, indentLv));
```

Comment:
Minor formatting changes here that don’t affect generation; these were just preexisting bugs that weren’t caught from the original lack of testing.